CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## Unreleased
  - Clarify documentation about macro indirection
  - Added changelog
+ - Turn the crate into a stdlib polyfill on rustc>=1.77
```

Comment:
This makes no sense. A polyfill is something that adds missing functionality. If anything this crate is a polyfill on *older* rustc. :)

Reply:
Oh, true - I was just reusing the wording from a bit further down.